src/dev/src/app/utils/demo-page.component.ts

```diff
@@ -0,0 +1,62 @@
+import { AfterContentInit, Component, ContentChildren, Input, QueryList } from '@angular/core';
+import { DemoComponent } from './demo.component';
+
+/**
+ * Class:  Demo-page.component.ts
+ *
+ * Description:
+ * ---------------------------------
+ * This class is building a Demo Page Component and takes as title as attribute.
```

Comment:
wording issue. "and takes as title as attribute" ???

Reply:
Fixed